### PR TITLE
remove webhook and secretshare alongside other singletons

### PR DESCRIPTION
**Pull request by bluzarraga**

### CODE REVIEW

File: isolate.sh [hunk 1]
```diff
@@ -552,6 +556,33 @@ function wait_for_condition() {
     fi
 }
 
+function cleanup_deployment() {
+    local name=$1
+    local namespace=$2
+    info "Deleting existing Deployment ${name} in namespace ${namespace}..."
+    ${OC} delete deployment ${name} -n ${namespace} --ignore-not-found
+
+    wait_for_no_pod ${namespace} ${name}
```
Comment (Daniel-Fan) [inline isolate.sh:565]: We might wanna remove this additional wait condition `wait_for_no_pod`, it is usually redundant.
If the deployment is removed, their pods should be gone as well

File: convert_to_multi_instance.sh [hunk 2]
```diff
@@ -686,6 +689,33 @@ function removeNSS(){
     success "Namespace Scope CRs cleaned up"
 }
 
+function cleanup_deployment() {
+    local name=$1
+    local namespace=$2
+    info "Deleting existing Deployment ${name} in namespace ${namespace}..."
+    ${OC} delete deployment ${name} -n ${namespace} --ignore-not-found
+
+    wait_for_no_pod ${namespace} ${name}
```
Comment (Daniel-Fan) [inline convert_to_multi_instance.sh:698]: ^^^

File: isolate.sh (hunk 1 above)
Comment (bluzarraga) [reply isolate.sh:565]: I'm going to remove it

**Comment by bluzarraga** [comment]
tested, should be ready to go

**Review by Daniel-Fan — APPROVED** [review]
/lgtm

**Pull request merged**